tests/skr/proxy.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fullstorydev/grpcurl:v1.9.1 as grpcurl-builder
+
+FROM python:latest
```

Comment:
I tried in this commit
https://github.com/microsoft/confidential-sidecar-containers/pull/117/commits/81bdd9e82a7b27d1ccabf1847895c63f21a13ab1

Is there any obvious mistake I'm making? Running the image locally works fine, but in ACI it almost immediately enters the terminated state
https://github.com/microsoft/confidential-sidecar-containers/actions/runs/9447436790/job/26019227329?pr=117

Reply:
There's a long standing issue with ACI where we can't exec into the mariner images and it enters the terminated state. This issue is why I haven't merged the conversion of ENCFS to Mariner. They are aware of it, but no ETA on when the fix will come out.

Reply:
That's interesting, I suspect this could be related but I don't think it's the same as I don't exec into the containers. I think while it's still not fully supported I would opt for sticking to the working base image

Reply:
That's fine by me.